x-pack/plugins/security_solution/public/management/pages/response_actions/view/response_actions_list_page.test.tsx

```diff
@@ -0,0 +1,284 @@
+      });
+
+      render();
+      const { getAllByTestId, getByTestId } = renderResult;
+      userEvent.click(getByTestId(`${testPrefix}-${filterPrefix}-popoverButton`));
+      const allFilterOptions = getAllByTestId(`${filterPrefix}-option`);
+
+      const selectedFilterOptions = allFilterOptions.reduce<string[]>((acc, option) => {
+        if (option.getAttribute('aria-checked') === 'true') {
+          acc.push(option.textContent?.split('-')[0].trim() as string);
+        }
+        return acc;
+      }, []);
+
+      expect(selectedFilterOptions.length).toEqual(2);
+      expect(selectedFilterOptions).toEqual(['Failed', 'Pending']);
+      expect(history.location.search).toEqual('?statuses=pending,failed');
+    });
+
+    // TODO: add tests for hosts and user when those filters are added
+
+    it('should read and set relative date ranges filter values from URL params', () => {
+      reactTestingLibrary.act(() => {
+        history.push('/administration/action_history?startDate=now-23m&endDate=now-1m');
+      });
+
+      render();
+      const { getByTestId } = renderResult;
+
+      expect(getByTestId('superDatePickerstartDatePopoverButton').textContent).toEqual(
+        '~ 23 minutes ago'
+      );
+      expect(getByTestId('superDatePickerendDatePopoverButton').textContent).toEqual(
+        '~ a minute ago'
+      );
+      expect(history.location.search).toEqual('?startDate=now-23m&endDate=now-1m');
+    });
+
+    // FIXME: the UI time should be local time
+    it.skip('should read and set absolute date ranges filter values from URL params', () => {
+      const startDate = '2022-09-12T11:00:00.000Z';
+      const endDate = '2022-09-12T11:30:33.000Z';
+      reactTestingLibrary.act(() => {
+        history.push(`/administration/action_history?startDate=${startDate}&endDate=${endDate}`);
+      });
+
+      const { getByTestId } = render();
+
+      expect(getByTestId('superDatePickerstartDatePopoverButton').textContent).toEqual(
+        'Sep 12, 2022 @ 13:00:00:000'
+      );
+      expect(getByTestId('superDatePickerendDatePopoverButton').textContent).toEqual(
+        'Sep 12, 2022 @ 13:30:33:000'
+      );
+      expect(history.location.search).toEqual('?startDate=now-23m&endDate=now-1m');
+    });
+  });
+
+  describe('Set selected/set values to URL params', () => {
+    // FIXME: select options by clicking
+    it.skip('should set selected command filter options to URL params ', () => {
+      const filterPrefix = 'actions-filter';
+      render();
+      const { getAllByTestId, getByTestId } = renderResult;
+      userEvent.click(getByTestId(`${testPrefix}-${filterPrefix}-popoverButton`));
+      const allFilterOptions = getAllByTestId(`${filterPrefix}-option`);
+
+      // FIXME
+      // test error:  unable to click element as it has or inherits pointer-events set to "none".
+      allFilterOptions.forEach((option) => userEvent.click(option));
```

Comment:
@paul-tavares this is where I'm struggling. Not sure what's the best way to select the filter options.

Reply:
I sually try to `.click()` them in the browwer using the dev tools console. Have you tried that?  I think we need to figure out what is actually the "clickable" - maybe guess is that maybe it is one of the parent elements of `option`.
If you are not able to figure it out, let me know tomorrow morning (my morning) and I'll see if I can figure it out


Also - maybe chdckout the EUi source code to see how they do it there. Could give you a clue

Reply:
So I looked into the eui lib and found that the click actually works for the inner span that has a class `euiSelectableListItem__content`. I tried that but jest complained about the same `pointer-events` set to `none` error. I went around it now with explicitly setting the pointer-events in the test to click the options. It worked.